### DISCUSSION TITLE
springboot/ioc

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#113] [Announcements]
# springboot/ioc

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/springboot/ioc.html

**Comment by chenzhuangwang** [comment]
二哥的这个spring也是讲的非常好，讲的清晰可以跟着直接上手，但是spring这一节还有springMVC没有看到在哪里欸，是spring学习阶段暂时不需要学习springmvc吗，springboot中直接上手吗好像太没明白
